action.yml

```diff
@@ -15,6 +15,12 @@ inputs:
 runs:
   using: 'composite'
   steps:
+    - name: Install GitHubActions.jl in its own (shared) environment
+      run: |
+        using Pkg
+        Pkg.activate("docs-logger-env"; shared=true)
```

Comment:
what do you mean by "use `JULIA_LOAD_PAT` env"?

Reply:
Seems to actually work: https://github.com/ericphanson/VisualStringDistances.jl/runs/3726621915?check_suite_focus=true

Reply:
Oh wow, was sure it was more recent.

> what do you mean by "use JULIA_LOAD_PAT env"?


```
JULIA_LOAD_PATH=.... julia -e 'using Pkg; ....'
```
or something

Reply:
ah yeah; the thing I was hoping to avoid by using the shared env's was figuring out writable directory in a known location to use in a cross-platform way. Definitely doable but can be annoying so I'm glad to just use shared envs for it.